chef-utils/lib/chef-utils/dsl/which.rb

```diff
@@ -78,7 +78,7 @@ def which(*cmds, extra_path: nil, &block)
       #
       def where(*cmds, extra_path: nil, &block)
         extra_path ||= __extra_path
-        paths = __env_path.split(File::PATH_SEPARATOR) + Array(extra_path)
+        paths = Array(extra_path) + __env_path.split(File::PATH_SEPARATOR)
```

Comment:
if you want to do this, add another kwarg prepend_path

Reply:
fixed in latest commit